frontend/packages/ux-editor/src/components/config/componentSpecificContent/Summary2/Override/Summary2OverrideEntry.tsx

```diff
@@ -29,6 +32,18 @@ export const Summary2OverrideEntry = ({
   const components = Object.values(formLayoutsData).flatMap((layout) =>
     getAllLayoutComponents(layout),
   );
+
+  const getComponentsInAGroup = () => {
+    return Object.values(formLayoutsData).flatMap((item) =>
+      Object.keys(item.order)
+        .filter((key) => key !== BASE_CONTAINER_ID)
+        .flatMap((key) => item.order[key]),
+    );
+  };
```

Comment:
I believe we can move this function out of the component to separate our code more. It can be put at the bottom of the file for example 😄

Reply:
fixed